lib/src/base-styles/_nx-colors.scss

```diff
@@ -136,8 +136,12 @@
   --nx-color-policy-indicator-text: var(--nx-color-text);
 
   //NxSmallThreatCounter
-  --nx-color-small-threat-counter-text: var(--nx-color-text-stark);
-  --nx-color-small-threat-counter-text-secondary: var(--nx-swatch-white);
+  --nx-color-small-threat-counter-critical-text: var(--nx-swatch-white);
+  --nx-color-small-threat-counter-severe-text: var(--nx-color-text-stark);
+  --nx-color-small-threat-counter-moderate-text: var(--nx-color-text-stark);
+  --nx-color-small-threat-counter-low-text: var(--nx-swatch-white);
+  --nx-color-small-threat-counter-none-text: var(--nx-color-text-stark);
+  --nx-color-small-threat-counter-unspecified-text: var(--nx-swatch-white);
```

Comment:
I was actually picturing these being separated from `NxSmallThreatCounter`, as they would be applicable anywhere text is being shown on a threat color background.  How about `--nx-color-threat-severe-text` etc?

Reply:
> I was actually picturing these being separated from `NxSmallThreatCounter`, as they would be applicable anywhere text is being shown on a threat color background. How about `--nx-color-threat-severe-text` etc?

Oops I misunderstood, that sounds good to me. Variable updated [4e577f4](https://github.com/sonatype/sonatype-react-shared-components/pull/1089/commits/4e577f461c2f37ab1883d01e1c811b1413af76b7), styles page updated [290ce09](https://github.com/sonatype/sonatype-react-shared-components/pull/1089/commits/290ce09a2639b891f23c0422d08c976fd6bd5faf)